### PR TITLE
Fix namespace mismatch metric

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward https://github.com/giantswarm/giantswarm/issues/13122

### CODE REVIEW

File: service/collector/namespace_mismatch.go [hunk 1]
```diff
@@ -70,11 +70,14 @@ func (n *NamespaceMismatch) Collect(ch chan<- prometheus.Metric) error {
...
 
-	var value float64
 	for _, chart := range charts.Items {
+		var value float64 = 0
+
 		content, err := n.helmClient.GetReleaseContent(ctx, chart.Spec.Name)
-		if err != nil {
-			n.logger.Log("level", "error", "message", "failed to collect namespace consistency", "stack", fmt.Sprintf("%#v", err))
+		if helmclient.IsReleaseNotFound(err) {
+			continue
```
Comment (tomahawk28) [inline service/collector/namespace_mismatch.go:78]: If the release is not found, keep continuing to other releases.

File: service/collector/namespace_mismatch.go [hunk 2]
```diff
@@ -70,11 +70,14 @@ func (n *NamespaceMismatch) Collect(ch chan<- prometheus.Metric) error {
...
+		} else if err != nil {
+			n.logger.Log("level", "warn", "message", "failed to collect namespace consistency", "stack", fmt.Sprintf("%#v", err))
```
Comment (tomahawk28) [inline service/collector/namespace_mismatch.go:80]: Change it to `warn` since it belongs to collector business.

**Review by rossf7 — APPROVED** [review]
LGTM

Should we have a fixed changelog?

**Comment by tomahawk28** [comment]
> Should we have a fixed changelog?

Good call, I'm adding it now.

**Pull request merged**